docs/fast_data/faq/debezium.md

```diff
@@ -0,0 +1,133 @@
+---
+id: debezium
+title: Debezium
+sidebar_label: Debezium
+---
+
+This page contains some useful answers to common questions about Debezium.
+
+# Debezium Connect
+This section covers the usage of Debezium as a Kafka Connect connector, specifically when using the official Docker image `quay.io/debezium/connect`.
+
+## How to filter messages before publishing to Kafka?
+To filter messages before puglishing to Kafka, you can use [Debezium Message Filtering](https://debezium.io/documentation/reference/2.6/transformations/filtering.html). Add the following properties to your connector's configuration:
+```
+...
+transforms=filter
+transforms.filter.type=io.debezium.transforms.Filter
+transforms.filter.language=jsr223.groovy
+transforms.filter.topic.regex=*.ingestion.MY_TABLE
+transforms.filter.condition=value.op == 'u' && value.before.id == 2
+...
+```
+For these properties to work, you need to add both `Debezium scripting` and `Groovy JSR` to the Debezium Docker image. You can do this by mounting a volume to the container or by building a new Docker image as follows:
+```Dockerfile
+FROM quay.io/debezium/connect:2.5
+
+# These versions to be used
+ENV DEBEZIUM_SCRIPTING_VERSION="2.5.4.Final" \
+    GROOVY_VERSION="4.0.21"
+
+# Download and extract Debezium scripting tar.gz
+RUN cd $KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-oracle && \
+    curl -LO https://repo1.maven.org/maven2/io/debezium/debezium-scripting/${DEBEZIUM_SCRIPTING_VERSION}/debezium-scripting-${DEBEZIUM_SCRIPTING_VERSION}.tar.gz && \
+    tar -xzf debezium-scripting-${DEBEZIUM_SCRIPTING_VERSION}.tar.gz && \
+    rm debezium-scripting-${DEBEZIUM_SCRIPTING_VERSION}.tar.gz
+
+# Download Groovy jars
+RUN cd $KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-oracle && \
+    curl -LO https://repo1.maven.org/maven2/org/apache/groovy/groovy/${GROOVY_VERSION}/groovy-${GROOVY_VERSION}.jar && \
+    curl -LO https://repo1.maven.org/maven2/org/apache/groovy/groovy-json/${GROOVY_VERSION}/groovy-json-${GROOVY_VERSION}.jar && \
+    curl -LO https://repo1.maven.org/maven2/org/apache/groovy/groovy-jsr223/${GROOVY_VERSION}/groovy-jsr223-${GROOVY_VERSION}.jar
+
+USER kafka
+```
+
+## Network errors on Debezium Logs
```

Comment:
this section looks more of a troubleshooting rather than a faq. 
is it better maybe to move some sections in the troubleshooting section?

Reply:
I have moved some paragraphs in the new section dedicated to Debezium Connect and I moved the remaining ones under the Troubleshooting section